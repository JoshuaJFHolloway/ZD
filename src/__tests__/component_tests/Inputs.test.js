import React from 'react';
import { shallow } from 'enzyme';
import Inputs from './../../components/Inputs';

const inputs = shallow(<Inputs />);

describe('Inputs', () => {

  it('renders correctly', () => {
    expect(Inputs).toMatchSnapshot();
  });

});