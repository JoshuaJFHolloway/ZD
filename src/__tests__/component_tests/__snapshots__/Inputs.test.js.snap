// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inputs renders correctly 1`] = `[Function]`;
